packages/bsky/src/views/index.ts

```diff
@@ -584,18 +590,22 @@ export class Views {
             like: viewer.like,
             threadMuted: viewer.threadMuted,
             replyDisabled: this.userReplyDisabled(uri, state),
+            embeddingDisabled: this.userPostEmbeddingDisabled(uri, state),
           }
         : undefined,
       labels,
       threadgate: !post.record.reply // only hydrate gate on root post
-        ? this.threadGate(gateUri, state)
+        ? this.threadgate(threadgateUri, state)
         : undefined,
     }
   }
 
   feedViewPost(
     item: FeedItem,
     state: HydrationState,
+    opts?: {
+      includeHiddenReplies?: boolean
+    },
```

Comment:
This OK? I want to maybe drill this into `replyRef` too, since it'd be nice to group the reply logic